packages/jest-runtime/src/index.js

```diff
@@ -496,6 +495,13 @@ class Runtime {
     localModule.parent = mockParentModule;
     localModule.paths = this._resolver.getModulePaths(dirname);
     localModule.require = this._createRequireImplementation(filename, options);
+    localModule.parent = Object.assign({}, localModule.parent, {
+      filename: lastExecutingModulePath,
+      require: this._createRequireImplementation(
```

Comment:
Couldn't this look up the parent module in `jest-runtime` and use its require here, instead of creating a new require implementation?

Reply:
If I can access the parent module, then that'd be perfect (as that's what `module.parent` is in node). I didn't find it though. Halp?

Reply:
we might be, I'll check